src/issue/screens/edit-issue-comment.screen.js

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { ScrollView, StyleSheet, TextInput, View } from 'react-native';
+import { ListItem } from 'react-native-elements';
+
+import { ViewContainer, SectionList, LoadingModal } from 'components';
+import { translate } from 'utils';
+import { colors, fonts, normalize } from 'config';
+import { editIssueComment } from '../issue.action';
+
+const styles = StyleSheet.create({
+  textInput: {
+    fontSize: normalize(12),
+    marginHorizontal: 15,
+    flex: 1,
+    color: colors.black,
+    ...fonts.fontPrimaryLight,
+  },
+  submitTitle: {
+    color: colors.green,
+    ...fonts.fontPrimary,
+  },
+  listItemContainer: {
+    flex: 1,
+  },
+  titleSmall: {
+    color: colors.primaryDark,
+    ...fonts.fontPrimarySemiBold,
+    fontSize: normalize(10),
+  },
+});
+
+const mapStateToProps = state => ({
+  language: state.auth.language,
+  repository: state.repository.repository,
+  isEditingComment: state.issue.isEditingComment,
+});
+
+const mapDispatchToProps = dispatch => ({
+  editIssueCommentByDispatch: (owner, repoName, issueTitle, issueComment) =>
+    dispatch(editIssueComment(owner, repoName, issueTitle, issueComment)),
+});
+
+class EditIssueComment extends Component {
+  props: {
+    editIssueCommentByDispatch: Function,
+    language: string,
+    repository: Object,
+    navigation: Object,
+    isEditingComment: boolean,
+    issue: Object,
+  };
+
+  state: {
+    issueComment: string,
+    issueCommentHeight: number,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      issueComment: this.props.navigation.state.params.comment.body,
+      issueCommentHeight: 0,
+    };
+  }
+
+  editIssueComment = () => {
+    const { navigation, editIssueCommentByDispatch } = this.props;
+    const { repository, comment } = this.props.navigation.state.params;
+
+    const repoName = repository.name;
+    const owner = repository.owner.login;
+    const text = this.state.issueComment;
+
+    editIssueCommentByDispatch(comment.id, owner, repoName, text).then(() =>
+      navigation.goBack()
+    );
+  };
+
+  render() {
+    const { language, repository, isEditingComment } = this.props;
+    const { issueComment } = this.state;
+    const issueTitle = this.props.navigation.state.params.issue.title;
+
+    return (
+      <ViewContainer>
+        {isEditingComment && <LoadingModal />}
+        <ScrollView>
+          {repository.full_name && (
+            <ListItem
+              title={`${repository.full_name} - ${issueTitle}`}
+              titleStyle={styles.titleSmall}
+              leftIcon={{
+                name: 'repo',
+                size: 17,
+                color: colors.grey,
+                type: 'octicon',
+              }}
+              hideChevron
+            />
+          )}
+
+          <SectionList
+            title={translate('issue.newIssue.issueComment', language)}
+          >
+            <TextInput
```

Comment:
This is great stuff. The only think I'm noticing is that with super long comments, this `TextInput` is massive

![screen shot 2017-09-22 at 2 22 54 pm](https://user-images.githubusercontent.com/12476932/30760292-907760a8-9fa7-11e7-8eec-993ec384bef5.png)

That blocks the `ScrollView` scroll capability and hides the the submit button. We'll need to have a fixed height for this input field (or a max height if that's possible - will be better if we can)

Reply:
One other thing is that it seems slightly squeezed vertically. 

![screen shot 2017-09-22 at 2 20 09 pm](https://user-images.githubusercontent.com/12476932/30760374-e29c4dda-9fa7-11e7-9d3b-911cfac725b6.png)

A little `paddingVertical` for the TextInput would be ideal :)